integration_test/pg/test_helper.exs

```diff
@@ -69,12 +69,15 @@ defmodule Ecto.Integration.Case do
   end
 end
 
+{:ok, _} = Application.ensure_all_started(:postgrex)
+
+Process.flag(:trap_exit, true)
+
 # Load up the repository, start it, and run migrations
-_   = Ecto.Storage.down(TestRepo)
+# _   = Ecto.Storage.down(TestRepo)
```

Comment:
This is commented out to get the tests passing in general. When it is enabled it will invariably result in:

```
** (MatchError) no match of right hand side value: {:error, %Postgrex.Error{message: "tcp recv: timeout", postgres: nil}}
```

The `storage_test` itself and the `ecto.create_drop_test` don't normally have such a timeout problem. I'm not sure what is causing the timeout, and could really use some help.